qiskit/primitives/base_sampler.py

```diff
@@ -184,7 +198,7 @@ def circuits(self) -> tuple[QuantumCircuit, ...]:
         Returns:
             The quantum circuits to be sampled.
         """
-        return self._circuits
+        return tuple(self._circuits)
```

Comment:
Now that circuits can be added to each `run()`, is this method intended to return the concatenated list of all circuits?

Reply:
Yes. I'm not sure that we need this methods in the future. Should we deprecate this?

Reply:
I guess wanting to see all circuits you submitted is a plausible use case? We already have to keep the circuits around so keeping this function shouldn't be a big deal.